# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: unknown
# Commit: 1d5a44ae7207
# Full Hash: 1d5a44ae72071f079dcb00b62419c2ddd3c9f12c
# Author: Norisz Fay <nfay@mozilla.com>
# Date: 2023-01-27 21:11 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing build bustages on DecoderData.cpp CLOSED TREE
# ==============================================================================

diff -r 83609f9575ac -r 1d5a44ae7207 third_party/rust/mp4parse/tests/irot-missing-essential.avif
Binary file third_party/rust/mp4parse/tests/irot-missing-essential.avif has changed